<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DummyControl::Form</class>
 <widget class="QWidget" name="DummyControl::Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QComboBox" name="comboBoxConnections"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEditData">
     <property name="text">
      <string>Hello world</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Data:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Send Injection</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditServiceId">
     <property name="text">
      <string>4095</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QPushButton" name="pushButtonMarker">
     <property name="text">
      <string>Marker</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="pushButtonQuit">
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditApplicationId">
     <property name="text">
      <string>LOG</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="pushButtonOpenFile">
     <property name="text">
      <string>OpenFile...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButtonClearFile">
     <property name="text">
      <string>ClearFile</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditContextId">
     <property name="text">
      <string>TEST</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="checkBoxAutoscroll">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Autoscroll</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="pushButtonDisconnect">
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ContextId:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushButtonSaveAsFile">
     <property name="text">
      <string>SaveAsFile...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>ApplicationId:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>ServiceId:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="pushButtonNewFile">
     <property name="text">
      <string>NewFile</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Jump To Msg with Index</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButtonJumpTo">
        <property name="text">
         <string>Jump to</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEditJumpTo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QPushButton" name="pushButtonReopenFile">
     <property name="text">
      <string>ReopenFile</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
